### PR TITLE
Conversation roles

### PR DESCRIPTION
**Pull request by marcoconti83**
## What's new in this PR?

This introduces conversation roles

### CODE REVIEW

File: WireSyncEngine.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -989,6 +1011,7 @@
 		87D4625C1C3D526D00433469 /* DeleteAccountRequestStrategyTests.swift */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.swift; path = DeleteAccountRequestStrategyTests.swift; sourceTree = "<group>"; };
 		87DF28C61F680495007E1702 /* PushDispatcherTests.swift */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.swift; path = PushDispatcherTests.swift; sourceTree = "<group>"; };
 		A9692F881986476900849241 /* NSString_NormalizationTests.m */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.c.objc; path = NSString_NormalizationTests.m; sourceTree = "<group>"; };
+		A984BB0023A3F027003B7204 /* WireDataModel.xcodeproj */ = {isa = PBXFileReference; lastKnownFileType = "wrapper.pb-project"; name = WireDataModel.xcodeproj; path = "../wire-ios-data-model/WireDataModel.xcodeproj"; sourceTree = "<group>"; };
```
Comment (billypchan) [inline WireSyncEngine.xcodeproj/project.pbxproj]: @marcoconti83 this is for debug only? The project should be restored.

File: WireSyncEngine.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -2595,7 +2658,6 @@
 				"$(SRCROOT)/Carthage/Build/iOS/WireUtilities.framework",
 				"$(SRCROOT)/Carthage/Build/iOS/libPhoneNumberiOS.framework",
 				"$(SRCROOT)/Carthage/Build/iOS/PINCache.framework",
-				"$(SRCROOT)/Carthage/Build/iOS/WireDataModel.framework",
```
Comment (billypchan) [inline WireSyncEngine.xcodeproj/project.pbxproj]: it should be restored.

File: WireSyncEngine.xcodeproj/project.pbxproj (hunk 1 above)
Comment (marcoconti83) [reply WireSyncEngine.xcodeproj/project.pbxproj]: Good catch, this is a mistake

**Comment by marcoconti83** [comment]
@zenkins release major

**Pull request merged**